### PR TITLE
makes cyro turn on automatically if you cyro yourself

### PR DESCRIPTION
**Pull request by ghost**
thanks to  JohnGinnane of hippiestation, this change is one i'm mirroring from hippie (they have some good ideas over there) and this...this is one of them.

### CODE REVIEW

**Comment by sawrge** [comment]
self serve medical :-1:

**Comment by PKPenguin321** [comment]
This was deliberately removed

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 1]
```diff
@@ -293,7 +293,8 @@
 			. = TRUE
 		if("door")
 			if(state_open)
-				close_machine()
+				if (close_machine() == usr)
+ 				on = 1
```
Comment (MrPerson) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: Those are SPACES you crazy fool!

**Comment by Bawhoppen** [comment]
EVERYTHING NEEDS TO BE STREAMLINED ANTAG SIMULATOR  !  !  !  !

**Comment by Robustin** [comment]
Does Hippie rebasing mean we have to endure terrible suggestions on a daily basis?

**Comment by RemieRichards** [comment]
No thank you.

**Pull request closed without merging**